predator/asp/greedy-target-seed-solving_SA.lp

```diff
@@ -25,53 +25,53 @@ metabolite(M) :- product(M,_).
 
 % try to cut the set of possible seeds
 can_reach(M) :- target(M).
-can_reach(M) :- reactant(M,R) ; product(P,R) ; can_reach(P).
+can_reach(M) :- reactant(M,R), product(P,R), can_reach(P).
```

Comment:
The semicolon is not necessary here.

Reply:
Unless colons and commas have different meaning in that context, I would personally avoid using comma as much as possible, since their use is tied to colons. Using semicolons by default makes writing of rules consistent across the file, whether or not colons are present.

Reply:
Commas are used to separate both literals in rule bodies as well as conditions in a conditional literal. To resolve this ambiguity, a condition is terminated with a semicolon ‘;’ (rather than ‘,’) when further body literals follow. In this case here  `reactant(M,R)` and `product(P,R)` are conditional literals without the condition (which are just regular body literals). So there is really no need for a semicolon.

Reply:
Just to clarify this PR is to discuss the encoding and not necessarily for merge. I also have first to figure out what's going on here. ;) As long as I got some quiet time.